src/components/home/GoogleMapContainer.js

```diff
@@ -13,7 +13,7 @@ export const GoogleMapContainer = (props) => {
   let markers = props.targets;
   return (
     <div className="map">
-      <Map center={location} markers={markers} updateTargetInfo={props.actions.updateFreeTarget} newTarget={props.newTarget} topicsList={props.topics}/>
+      <Map center={location} markers={markers} updateTargetInfo={props.actions.updateFreeTarget} newTarget={props.newTarget} topicsList={props.topics} />
```

Comment:
When the line gets too long try to split it.
With this format: https://github.com/airbnb/javascript/tree/master/react#alignment